src/mono/mono/mini/mini-arm64.c

```diff
@@ -2209,8 +2209,14 @@ mono_arch_allocate_vars (MonoCompile *cfg)
 		cfg->ret->dreg = cinfo->ret.reg;
 		break;
 	case ArgVtypeInIRegs:
-	case ArgHFA:
+	case ArgHFA: {
 		/* Allocate a local to hold the result, the epilog will copy it to the correct place */
+		MonoType *ret_type = mini_get_underlying_type (sig->ret);
+		if (MONO_CLASS_IS_SIMD (cfg, mono_class_from_mono_type_internal (ret_type))) {
+			int align = 16;
```

Comment:
Should this be handling `Vector64<T>` which has a different alignment requirement?

Reply:
This is just a local variable which stores the result, it can be over aligned.